### DISCUSSION TITLE
Conventions for determining what/whether something warrants working on?

### DISCUSSION THREAD

**Discussion by brucearctor** [pixeltable/pixeltable#313] [Q&A]
Does this community have established/preferences for how to suggest work that gets done, and whether that gets done [ even if aiming to do oneself ]?

Unclear whether to submit GH Issue, or if preference is to submit Q&A.

**Comment by pierrebrunelle** [comment]
We added a contributor guide: https://github.com/pixeltable/pixeltable/blob/main/CONTRIBUTING.md that answers the above questions.
